[2023-08-12T00:09:20.054+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_w_postgres_v3.create_table_in_postgres scheduled__2023-08-11T00:00:00+00:00 [queued]>
[2023-08-12T00:09:20.061+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_w_postgres_v3.create_table_in_postgres scheduled__2023-08-11T00:00:00+00:00 [queued]>
[2023-08-12T00:09:20.061+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-08-12T00:09:20.076+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): create_table_in_postgres> on 2023-08-11 00:00:00+00:00
[2023-08-12T00:09:20.082+0000] {standard_task_runner.py:57} INFO - Started process 83724 to run task
[2023-08-12T00:09:20.090+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_w_postgres_v3', 'create_table_in_postgres', 'scheduled__2023-08-11T00:00:00+00:00', '--job-id', '1134', '--raw', '--subdir', 'DAGS_FOLDER/dag_postgresql.py', '--cfg-path', '/tmp/tmp82bcta3b']
[2023-08-12T00:09:20.101+0000] {standard_task_runner.py:85} INFO - Job 1134: Subtask create_table_in_postgres
[2023-08-12T00:09:20.173+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_w_postgres_v3.create_table_in_postgres scheduled__2023-08-11T00:00:00+00:00 [running]> on host e3ce6cb94b48
[2023-08-12T00:09:20.388+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Zack' AIRFLOW_CTX_DAG_ID='dag_w_postgres_v3' AIRFLOW_CTX_TASK_ID='create_table_in_postgres' AIRFLOW_CTX_EXECUTION_DATE='2023-08-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-11T00:00:00+00:00'
[2023-08-12T00:09:20.395+0000] {sql.py:265} INFO - Executing: 
            CREATE TABLE if not exists dag_runs(
                dt date,
                dag_id character varying,
                primary key  (dt, dag_id)
            )
        
[2023-08-12T00:09:20.446+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-08-12T00:09:20.513+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-08-12T00:09:20.535+0000] {sql.py:375} INFO - Running statement: 
            CREATE TABLE if not exists dag_runs(
                dt date,
                dag_id character varying,
                primary key  (dt, dag_id)
            )
        , parameters: None
[2023-08-12T00:09:20.547+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_w_postgres_v3, task_id=create_table_in_postgres, execution_date=20230811T000000, start_date=20230812T000920, end_date=20230812T000920
[2023-08-12T00:09:20.582+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-12T00:09:20.603+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
